src/theory/tools/wp/bir_htScript.sml

```diff
@@ -0,0 +1,614 @@
+open HolKernel Parse boolLib bossLib;
+
+(* From /core: *)
```

Comment:
This path changed, it should be `theory/bir` now. Same for `/theories` => `theory/bir-support.`.

Isn't `HolBACoreSimps` in `theory/bir`?

Reply:
> This path changed, it should be `theory/bir` now. Same for `/theories` => `theory/bir-support.`.

Check, will be addressed in a new commit.

> Isn't `HolBACoreSimps` in `theory/bir`?

Yes. All the theories, function libraries, syntax function libraries and simplification set libraries are also sorted by category, although this is hard to see from this example. I will write better comments so that this can be immediately seen.

By the way, I don't think it's necessary to always sort imports in this way, it's just tidy when you have 15+ of them.